test/betting/test.betFinalize.mligo

```diff
@@ -0,0 +1,238 @@
+#import "../../src/contracts/cameligo/betting/errors.mligo" "Errors"
+#import "helpers/bootstrap.mligo" "Bootstrap"
+#import "helpers/helper.mligo" "Helper"
+#import "helpers/assert.mligo" "Assert"
+#import "helpers/events.mligo" "Events"
+#import "helpers/log.mligo" "Log"
+
+let () = Log.describe("[Betting - FinalizeBet] test suite")
+
+(**
+ *  Basic Win Reward Test
+ *)
+let test_win_basic_team1_should_work = 
+  //Given
+  let (_, betting_contract, betting_taddress, elon, _, alice, bob, mike) = Bootstrap.bootstrap() in
+  let () = Helper.trsc_add_event_success (betting_contract, elon, Events.eventype_to_addeventparam(Events.primary_event)) in
+  let () = Helper.trsc_add_bet_success (betting_contract, alice, 0n, (true  : bool), 800000000000mutez) in
+  let () = Helper.trsc_add_bet_success (betting_contract, bob,   0n, (true  : bool), 800000000000mutez) in
+  let () = Helper.trsc_add_bet_success (betting_contract, mike,  0n, (false : bool), 800000000000mutez) in
+  let () = Helper.trsc_update_event_success (betting_contract, elon, 0n, Events.finalized_event_team1_win) in
+  let storage = Test.get_storage(betting_taddress) in
+  let quota_left : nat = abs(100n - storage.bet_config.retained_profit_quota) in
+  let expected_alice_balance = Test.get_balance(alice) + (800000000000mutez + 400000000000mutez) * quota_left / 100n in
+  let expected_bob_balance   = Test.get_balance(bob)   + (800000000000mutez + 400000000000mutez) * quota_left / 100n in
+  let expected_mike_balance  = Test.get_balance(mike) in
+  //When
+  let () = Helper.trsc_finalize_bet_success (betting_contract, elon, 0n) in
+  //Then
+  let () = assert (expected_alice_balance = Test.get_balance(alice)) in
+  let () = assert (expected_bob_balance   = Test.get_balance(bob)) in
+  let () = assert (expected_mike_balance  = Test.get_balance(mike)) in
```

Comment:
May you could have used `Assert.balance`

Reply:
I could not find it in the doc, do you have a link ?

Reply:
I was thinking about using this function https://github.com/frankhillard/predictive-market-cameligo/blob/initial_draft/test/betting/helpers/assert.mligo#L21-L23

Reply:
Done, thanks Melwyn 👍🏻